src/DDoSX/CdnClient.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\Client;
+use UKFast\SDK\DDoSX\Entities\Cdn;
+use UKFast\SDK\SelfResponse;
+
+class CdnClient extends Client
+{
+    /**
+     * @inheritDoc
+     */
+    protected $basePath = 'ddosx/';
+
+    /**
+     * @param Cdn $cdn
+     * @return SelfResponse
+     */
+    public function create(Cdn $cdn)
+    {
+        $response = $this->post(
+            'v1/domains/' . $cdn->name . '/cdn',
+            json_encode($this->friendlyToApi($cdn, $this->requestMap))
+        );
+        $body = $this->decodeJson($response->getBody()->getContents());
+
+        return (new SelfResponse($body, "domain_name"))
+            ->setClient($this)
+            ->serializeWith(function ($response) {
+                return new Cdn($this->apiToFriendly($response->data[0], $this->requestMap));
```

Comment:
Where is requestMap declared in this client class? It is declared in the admin class

Reply:
Updated it :+1: